examples/mysql-test.yml

```diff
@@ -11,6 +11,110 @@ connect:
   user: "${DB_USER}"
   password: "${DB_PASSWORD}"
 
+actions:
```

Comment:
We should probably test the update user attr action in the postgres CI test.

It would be nice if we ran the mysql example in CI, similar to how we run the postgres example. But that can be a separate pull request.